content/about/index.md

```diff
@@ -1,10 +1,30 @@
 ---
+## Configure header of page
 title: "About"
-description: A blog template for Hugo developed by Formspree and available for free.
+description: "A blog template for Hugo developed by Formspree and available for free."
+author: "Emily Cooper"
+headline: "Hi, my name is Emily Cooper.<br> Nice to meet you."
```

Comment:
```suggestion
headline: |
  Hi, my name is Emily Cooper.
  Nice to meet you."
```

not sure about the number of space needed before the lines. I was just thinking it might be nice to not have the br?

Reply:
Looks like it still needs a `<br>` to create the line break though

Reply:
right or maybe one more empty line?